packages/dev/core/src/scene.ts

```diff
@@ -219,10 +219,28 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
      * Gets or sets a boolean that indicates if the scene must clear the depth and stencil buffers before rendering a frame
      */
     public autoClearDepthAndStencil = true;
+
+    private _clearColor: Color4 = new Color4(0.2, 0.2, 0.3, 1.0);
+
+    /**
+     * Observable triggered when the performance priority is changed
+     */
+    public onClearColorChangedObservable = new Observable<Color4>();

```

Comment:
Is this added for consistency? wondering what would use it knowing we only check the color reference to trigger it ?

Reply:
Since there is a `clearColor` property (`clear-color` attribute) on the Viewer (which when set just sets the clear color of the scene), I wanted this observable so I could keep the properties in sync. It's not critical, I just think it makes more sense.